llvm/lib/CodeGen/ValueTypes.cpp

```diff
@@ -161,6 +161,12 @@ TypeSize EVT::getExtendedSizeInBits() const {
 std::string EVT::getEVTString() const {
   switch (V.SimpleTy) {
   default:
+    if (isRISCVVectorTuple()) {
+      unsigned Sz = getSizeInBits();
+      unsigned NF = getRISCVVectorTupleNumFields();
+      unsigned MinNumElts = Sz / (NF * 8);
+      return "riscv_nxv" + utostr(MinNumElts) + "i8";
```

Comment:
Shouldn't there be a `xNF` at the end to match the names in ValueTypes.td?

Reply:
Correct~